selftests/checkall

```diff
@@ -5,6 +5,7 @@ run_rc() {
     eval $1
     if [ $? != 0 ]; then
         GR=1
+        [ ! "$SELF_CHECK_CONTINUOUS" ] && exit 1
```

Comment:
`make check` will fail earlier if the error is in one of the steps (one of the run_rc() calls), but it will not fail fast during the selftests. To do that, you have to add `failfast=True` as argument to `unittest.TextTestRunner()` in selftests/run. Also, add the `--failfast` option to the avocado self check command.